temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/IvyModal.kt

```diff
@@ -299,6 +299,8 @@ fun ModalActionsRow(
                 onClick = onClose
             )
 
+            Spacer(Modifier.weight(1f))
```

Comment:
Can you put that in temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt in your `if`?

Applying this change to all Ivy modals may break some existing ones.